app/src/main/java/com/hedvig/app/ApplicationModule.kt

```diff
@@ -476,3 +478,10 @@ val useCaseModule = module {
     single { StartNorwegianAuthUseCase(get()) }
     single { SubscribeToAuthStatusUseCase(get()) }
 }
+
+val offerPersistenceManagerModule = module {
+    single {
+        val sharedPreferences = PreferenceManager.getDefaultSharedPreferences(get())
```

Comment:
Should we let `Koin` instantiate this dependency instead? If we do that, the instance can be reused between dependencies, now we're requiring one to be created specifically for `OfferPersistenceManager` 🤔